transport/src/main/java/io/netty/channel/embedded/EmbeddedChannel.java

```diff
@@ -338,6 +350,14 @@ public boolean writeInbound(Object... msgs) {
             return isNotEmpty(inboundMessages);
         }
 
+        if (!config().isAutoRead()) {
+            for (Object m : msgs) {
+                tempInboundMessages().add(m);
+            }
+            // write in buffer, return true, too.
+            return true;
```

Comment:
imho that is not correct. We should return false as we did not produce any message ?

Reply:
fixed...
I misunderstood the comments of this method..